frontend/store/projects.js

```diff
@@ -160,7 +160,7 @@ export const getters = {
       text: 'CSV'
     }
     const json = {
-      type: 'json',
+      type: 'json1',
```

Comment:
should be jsonl with an "L"

Reply:
(below too)

Reply:
Should I change it completely? Since in the codebase(see for example views.py line 377) its also defined as 'json1'.

Reply:
I think it's better to create a separate pull for this, since it would be out of scope here.